java/src/test/java/org/rocksdb/RocksDBTest.java

```diff
@@ -602,6 +602,60 @@ public void compactRangeToLevel()
     }
   }
 
+  @Test
+  public void deleteFilesInRange()
+      throws RocksDBException, InterruptedException {
+    final int KEY_SIZE = 20;
+    final int VALUE_SIZE = 3000;
+    final int FILE_SIZE = 100000;
+    final int NUM_FILES = 10;
+    
+    final int KEY_INTERVAL = 10000;
+    try (final Options opt = new Options().
+        setCreateIfMissing(true).
+        setCompactionStyle(CompactionStyle.LEVEL).
+        setCompressionType(CompressionType.NO_COMPRESSION).
+        setTargetFileSizeBase(FILE_SIZE).
+        setTargetFileSizeMultiplier(1).
+        setWriteBufferSize(FILE_SIZE / 2).
+        setNumLevels(5).
+        setDisableAutoCompactions(true);
+         final RocksDB db = RocksDB.open(opt,
+             dbFolder.getRoot().getAbsolutePath())
+    ) {
+      int records = FILE_SIZE / (KEY_SIZE + VALUE_SIZE);
+
+      // fill database with key/value pairs
+      byte[] value = new byte[VALUE_SIZE];
```

Comment:
Could we show the same results with less key value pairs and less files? Just thinking about execution time...

Reply:
Execution time is 1 second. I was trying to make files smaller, but compactions are not predicable much. This combination is stable. I will try to squeeze that little more.

Reply:
1 second is fine. I thought from the options that auto compactions were disabled? Maybe I am not understanding something, I have not worked a lot with testing around compactions